### DISCUSSION TITLE
Are Datomic subscriptions supported?

### DISCUSSION THREAD

**Discussion by faceyspacey** [day8/re-frame#698] [Q&A]
David Nolen, in one of his videos, claims that Om Next supports subscriptions via its Relay-style query interface. But I haven't been able to find any examples of such. I'm newly a Javascript refufee (coming from Meteor and turning to CLJS), and Re-Frame seems like my new god lol, but I'm worried that it hasn't pursued the Relay-influenced features as much as Om Next. 

How do you go about go about Datomic queries and mutations from Re-Frame components and are the queries reactive subscriptions (i.e. they re-render components when server side Datomic state changes)?

**Comment by mike-thompson-day8** [comment]
@faceyspacey this is not the place to ask for support.  Repo issues are for bugs and feature requests. For support, please use http://clojurians.net/  or newsgroups like  https://groups.google.com/forum/#!forum/clojurescript
